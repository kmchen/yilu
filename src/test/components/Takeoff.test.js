import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Takeoff from '../../components/TakeOff';
import data from '../../../data/data.js';

configure({ adapter: new Adapter() });

describe('Takeoff component', () => {
  let takeoffComponent;

  beforeEach(() => {
    const props = {
      airportName: 'some name',
      busynessIndex: 10
    };
    takeoffComponent = shallow(<Takeoff {...props} />);
  });

  it('should render Landing component', () => {
    expect(takeoffComponent).toMatchSnapshot();
  });
});

