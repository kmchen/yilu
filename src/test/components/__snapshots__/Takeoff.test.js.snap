// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Takeoff component should render Landing component 1`] = `
<WithStyles(ForwardRef(Grid))
  alignItems="center"
  container={true}
  spacing={1}
>
  <WithStyles(ForwardRef(Grid))
    item={true}
    xs={4}
  >
    <WithStyles(ForwardRef(Avatar))
      className="makeStyles-avatar-1"
    >
      <FlightTakeoffIcon />
    </WithStyles(ForwardRef(Avatar))>
  </WithStyles(ForwardRef(Grid))>
  <WithStyles(ForwardRef(Grid))
    item={true}
    xs={8}
  >
    <WithStyles(ForwardRef(Typography))>
      some name Airport
    </WithStyles(ForwardRef(Typography))>
    <Styled(MuiBox)
      display="flex"
      justifyContent="space-between"
    >
      <Styled(MuiBox)
        color="text.primary"
      >
        Terminal C
      </Styled(MuiBox)>
      <Styled(MuiBox)
        className="makeStyles-traffic-2"
      >
        buzy
      </Styled(MuiBox)>
    </Styled(MuiBox)>
  </WithStyles(ForwardRef(Grid))>
</WithStyles(ForwardRef(Grid))>
`;
